### PR TITLE
Applied code formatting & cleanup

### PR DESCRIPTION
**Pull request by alxsap**
Formatting all code to have a clean state to continue.

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java [hunk 1]
```diff
@@ -91,21 +88,22 @@ static void parse(byte[] b, int len, Map<String, Object> data, boolean saveAnnot
...
+                data.put(ANNOTATIONS_KEY, annotations);
             }
         }
     }
 
     static void write(Writer w, Map<String, Object> source) throws IOException {
-        if(source == null) throw new NullPointerException("source is null");
-        String message = (String)source.get(MESSAGE_KEY);
+        if (source == null)
+            throw new NullPointerException("source is null");
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java]: This should also be changed. Either single line or curly braces would be my recommendation.

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchAccess.java [hunk 2]
```diff
@@ -46,192 +46,157 @@
...
  *
  */
-public class ElasticSearchAccess
-{
-  private static final Logger LOGGER = Logger.getLogger(ElasticSearchAccess.class.getName());
+public class ElasticSearchAccess {
+    private static final Logger LOGGER = Logger.getLogger(ElasticSearchAccess.class.getName());
 
-  private static final Range<Integer> SUCCESS_CODES = closedOpen(200, 300);
+    private static final Range<Integer> SUCCESS_CODES = closedOpen(200, 300);
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchAccess.java]: 4 blanks? 2 blanks? 4 blanks are better readable but cause longer lines.
Prominent Jenkins projects use 4 blanks.

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java [hunk 3]
```diff
@@ -67,11 +62,13 @@ static void parse(byte[] b, int len, Map<String, Object> data, boolean saveAnnot
...
             }
         }
-        
+
         String line = new String(b, 0, eol, StandardCharsets.UTF_8);
-        
-        // Would be more efficient to do searches at the byte[] level, but too much bother for now,
-        // especially since there is no standard library method to do offset searches like String has.
+
+        // Would be more efficient to do searches at the byte[] level, but too much
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java]: I would increase max. line width (from 80) to 120. What is the recommendation by GitHub for correct display?

Comment (alxsap) [reply src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java]: In GitHub up to 140 characters are visible per line.

**Comment by MichaelSp** [comment]
I just did the same with Intellij 2019 and it [looks differently](https://github.com/SAP/elasticsearch-logs-plugin/compare/PR_format...MichaelSp:PR_format)

most notable:
   * Imports are ordered alphabetically
  * methods are moved around (not sure which pattern; also alphabetically?)
  * formatting in HTML, too

I've also added this [.editorconfig](https://github.com/MichaelSp/elasticsearch-logs-plugin/blob/PR_format/.editorconfig)

**Comment by alxsap** [comment]
@MichaelSp 
- Editorconfig is a good idea. Unfortunately only very few rules are possible there 😕

- > Imports are ordered alphabetically

  would be OK

- > methods are moved around (not sure which pattern; also alphabetically?)

  this I don't like 

- > formatting in HTML, too

  I only formatted java files. Maybe 2 spaces indent for HTML?

In general, from my perspective, this reformatting should be a one time action (or after some time again). Files should not be reformatted every time a file is saved. It is very unlikely that every contributer really has exactly the same formatting rules. Especially if configuration is manual effort in different IDEs used.

In Eclipse I remember we configured to only format the changed lines. With full reformatting you also don't make friends in projects you contribute to.

**Comment by MichaelSp** [comment]
Fully agreed. 
I wouldn't want to review contributions with all that noise.

I've updated [my branch](https://github.com/SAP/elasticsearch-logs-plugin/compare/PR_format...MichaelSp:PR_format) without moving methods around.

**Comment by alxsap** [comment]
Hi @MichaelSp, would this be OK for you?
https://github.com/SAP/elasticsearch-logs-plugin/pull/9

Import sorting is not completely alphabetic in Eclipse and also seems not recommended.

**Comment by alxsap** [comment]
https://github.com/SAP/elasticsearch-logs-plugin/pull/9 merged

**Pull request closed without merging**